packages/experience-builder-sdk/src/blocks/preview/CompositionBlock.tsx

```diff
@@ -263,14 +266,17 @@ export const CompositionBlock = ({
     );
   }
 
-  //List explicit style props that will end up being passed to the component
-  const stylesToKeep = ['cfImageAsset'];
-  const stylesToRemove = CF_STYLE_ATTRIBUTES.filter((style) => !stylesToKeep.includes(style));
+  if (
+    node.definitionId === CONTENTFUL_COMPONENTS.image.id &&
+    node.variables.cfImageAsset?.type === 'UnboundValue'
+  ) {
+    return <PreviewUnboundImage node={node} nodeProps={nodeProps} component={component} />;
+  }
 
   return React.createElement(
     component,
     {
-      ...omit(nodeProps, stylesToRemove, ['cfHyperlink', 'cfOpenInNewTab', 'cfSsrClassName']),
+      ...sanitizeNodeProps(nodeProps),
```

Comment:
Will this fix the errors we get about unknown props being spread onto html elements that have popped up again?

Reply:
Probably not, unfortunately. I didn't actually change this code, I just moved it to a util file to make it reusable.  We had this same code duplicated in 2 files and I needed to do the same thing in 3rd file.  But at least now there is a single source of these bits instead of multiple copies.